pages/developers/gha_wheels.md

```diff
@@ -95,14 +103,14 @@ The core of the work is down here:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-18.04, windows-latest, macos-latest]
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
```

Comment:
Why not keep "latest" and rather pin certain versions?

Reply:
It's pretty close. Pinning versions gets rid of the "this will change soon" warning on all your builds for ubuntu-latest. The change to macOS-11.0 will likely be something intentional, since currently you would lose Python 3.5 and PyPy if it were to change today. And the difference between windows-2017 and windows-2019 was that you might have to add an extra line to your setup.py to not pick up an extra dependency, which Matplotlib found out the hard way. So having it stable is likely a _hair_ better. Though we will try to keep the issues down to a minimum upstream by the time the default changes, so `-latest` is not bad either - and maybe it's better so we don't have to go manually update the pins? Currently it's 5 or so packages, which isn't awful, but it could grow....

Reply:
Thanks @henryiii for the comprehensive and clear explanations. Makes sense.